contracts/StakingVe.sol

```diff
@@ -0,0 +1,305 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
+
+/// @title A vesting style staking contract with extendable linear decay
+/// @author Auroter
+/// @notice Allows you to lock tokens in exchange for governance tokens
+/// @notice Locks can be extended or deposited into
+/// @notice Maximum deposit duration is two years (104 weeks)
+/// @dev If IsUserLocked is false, call CreateLock
+/// @dev If IsUserLocked is true, call ExtendMyLock
+
+// We have to maintain old versions of this to use legacy solidity (0.6.12)
+// Otherwise it's safe to link with @OpenZeppelin/
+import "./libraries/SafeERC20.sol";
+import "./libraries/SafeMath.sol";
+import "./interfaces/IERC20.sol";
+
+//import "hardhat/console.sol";
+
+contract StakingVe {
+    using SafeMath for uint256;
+    using SafeERC20 for IERC20;
+
+    // Info pertaining to staking contract
+    address private StakedToken; // An ERC20 Token to be staked (i.e. Axial)
+    address private Governance; // Who has administrative access
+    string private Name; // New asset after staking (i.e. sAxial)
```

Comment:
What is the use of this variable and Symbol? They are both private and assigned in the constructor but never used.